test/getHashDigest.test.js

```diff
@@ -11,8 +11,8 @@ describe("getHashDigest()", () => {
     ["abc\\0💩", "xxhash64", "hex", undefined, "86733ec125b93904"],
     ["abc\\0💩", "xxhash64", "base64", undefined, "hnM+wSW5OQQ="],
     ["abc\\0♥", "xxhash64", "base64", undefined, "S5o0KX3APSA="],
-    ["abc\\0💩", "xxhash64", "base52", undefined, "cfByjQcJZIU"],
-    ["abc\\0♥", "xxhash64", "base52", undefined, "qdLyAQjLlod"],
+    ["abc\\0💩", "xxhash64", "base52", undefined, "acfByjQcJZIU"],
+    ["abc\\0♥", "xxhash64", "base52", undefined, "aqdLyAQjLlod"],
```

Comment:
Here, now the diff is much cleaner and it's more clear what happens.

In this case the hash was missing the leading "a" (most significant zeroes are now preserved). Now it's 12 chars long.

Please see
```
["test string", "xxhash64", "base52", undefined, "byfYGDmnmyUr"]
```
above for comparison. It was always 12 chars long and is unchanged in this PR.